### DISCUSSION TITLE
I can't run it

### DISCUSSION THREAD

**Discussion by naza1144** [Rapptz/discord.py#7773] [Q&A] [answered]
![Screenshot 2022-03-27 140338](https://user-images.githubusercontent.com/85209746/160270613-95da0fd5-153f-4608-bb7c-2398e2f654f9.png)
![Screenshot 2022-03-27 140401](https://user-images.githubusercontent.com/85209746/160270616-c2b65206-9b9a-4a93-9b8e-06aeb5643dee.png)

**Comment by ChesterWOV-DPY** [comment, marked as answer]
According to the dpy tag `?tag wmc`:
### **Why does my bot doesn't respond to commands in v2.0? / Why is `message.content` not there.**
Discord.py v2.0 has migrated to api v10, and with this has come the introduction of `Intents.message_content`, which is **disabled by default**.
This means that you will need to __explicitly enable it in code and in the panel, at <https://discord.com/developers/applications>.__
```py
intents = discord.Intents.default()  # All but the THREE privileged ones
intents.message_content = True  # Subscribe to the Message Content intent
bot = commands.Bot(command_prefix='-', intents=intents)  # Pass the intents into your commands.Bot or discord.Client
```
NOTE: Still not responding to commands? see https://discordpy.readthedocs.io/en/latest/faq.html#why-does-on-message-make-my-commands-stop-working

**Reply by naza1144** [reply]
ummm I not understand sorry.  Me new I just started writing bot is 2 day.

**Reply by naza1144** [reply]
you can show how to can run it

**Reply by ChesterWOV-DPY** [reply]
Are you using 2.0? If you are, enable the message_content intents as shown in the example. If you aren't and you don't have an `on_message` event, tell me.

**Reply by naza1144** [reply]
![image](https://user-images.githubusercontent.com/85209746/160286889-98451848-3700-402f-8e23-49c300abf092.png)

That's right?

**Comment by jelni** [comment]
Does your code have `bot.run(TOKEN)` anywhere? Did you show us the full code?

**Reply by naza1144** [reply]
![image](https://user-images.githubusercontent.com/85209746/160312856-679de8fa-86da-4f39-8bf7-7dc97a73b0a4.png)

**Reply by dartmern** [reply]
You have 2 instances of bots/clients. Use either commands.Bot or discord.Client. Your adding the command to your "bot" instance and running your bot with the "client" instance.

**Reply by ChesterWOV-DPY** [reply]
Just wanted to say that. True

> You have 2 instances of bots/clients. Use either commands.Bot or discord.Client. Your adding the command to your "bot" instance and running your bot with the "client" instance.

**Reply by naza1144** [reply]
ok👍
